src/admin.js

```diff
@@ -2,18 +2,19 @@
 /* eslint no-labels: ['error', { 'allowLoop': true }] */
 import 'fetch-everywhere';
 import StitchClient from './client';
-import ADMIN_TYPE from './common';
+import ADMIN_CLIENT_TYPE from './common';
+import { ADMIN_CLIENT_CODEC } from './auth/common';
 
 const v1 = 1;
 const v2 = 2;
 
 export default class Admin extends StitchClient {
   constructor(baseUrl) {
-    super('', {baseUrl});
+    super('', {baseUrl, authCodec: ADMIN_CLIENT_CODEC});
```

Comment:
I don't think this is the way to do this. I think ideally Admin sets up its own `Auth` by doing `this.auth` or you call super with a `authConfig` with contains `url` and `codec`.